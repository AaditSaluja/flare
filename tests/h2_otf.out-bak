2019-09-18 17:46:26.491841
number of cpu cores: 4
cutoffs: [5]
kernel: two_body
number of hyperparameters: 3
hyperparameters: [1 1 1]
hyperparameter optimization algorithm: L-BFGS-B
uncertainty tolerance: 0.1 eV/A
timestep (ps): 0.0001
number of frames: 20
number of atoms: 2
system species: {'H'}
periodic cell: 
[[5. 0. 0.]
 [0. 5. 0.]
 [0. 0. 5.]]
previous positions (A):
H 2.30000000 2.50000000 2.50000000 
H 2.80000000 2.50000000 2.50000000 
--------------------------------------------------------------------------------

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 1 
wall time from start: 2.35 s 
