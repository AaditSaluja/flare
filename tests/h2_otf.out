2019-03-28 00:40:27.376502
number of cpu cores: 4
cutoffs: [5]
kernel: two_body
number of hyperparameters: 3
hyperparameters: [1 1 1]
hyperparameter optimization algorithm: L-BFGS-B
uncertainty tolerance: 0.1 eV/A
timestep (ps): 0.0001
number of frames: 20
number of atoms: 2
system species: {'H'}
periodic cell: 
[[5. 0. 0.]
 [0. 5. 0.]
 [0. 0. 5.]]
previous positions (A):
H 2.30000000 2.50000000 2.50000000 
H 2.80000000 2.50000000 2.50000000 
--------------------------------------------------------------------------------

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 1 
wall time from start: 0.31 s 

*-Frame: 0
Simulation Time: 0.000 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.30000000 2.50000000 2.50000000 	-22.29815461 0.00000000 0.00000000 	0.00000000e+00 0.00000000e+00 0.00000000e+00 	-1.07572244e+01 0.00000000e+00 0.00000000e+00 
H 2.80000000 2.50000000 2.50000000 	22.29815461 0.00000000 0.00000000 	0.00000000e+00 0.00000000e+00 0.00000000e+00 	1.07572244e+01 0.00000000e+00 0.00000000e+00 

temperature: 46.39 K 
kinetic energy: 0.011993 eV 
potential energy: 0.000000 eV 
total energy: 0.011993 eV 
wall time from start: 0.31 s 

Adding atom 0 to the training set.
Uncertainty: [0. 0. 0.].

GP hyperparameters: 
Hyp0 : Signal Std = 1.003045609360523
Hyp1 : Length Scale = 0.9975383300189093
Hyp2 : Noise Std = 1e-06
likelihood: 21.269701591854922
likelihood gradient: [-9.90859427e-05  8.29682155e-05 -2.00000000e+06]
wall time from start: 3.05 s 
--------------------------------------------------------------------------------
-Frame: 1
Simulation Time: 0.000 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.29784856 2.50000000 2.50000000 	-22.25905884 0.00000000 0.00000000 	1.62346655e-01 0.00000000e+00 0.00000000e+00 	-3.22528122e+01 0.00000000e+00 0.00000000e+00 
H 2.80215144 2.50000000 2.50000000 	22.25905884 0.00000000 0.00000000 	1.62346655e-01 0.00000000e+00 0.00000000e+00 	3.22528122e+01 0.00000000e+00 0.00000000e+00 

temperature: 417.04 K 
kinetic energy: 0.107814 eV 
potential energy: 8.114370 eV 
total energy: 8.222184 eV 
wall time from start: 3.48 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 2 
wall time from start: 3.79 s 

*-Frame: 1
Simulation Time: 0.000 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.29784856 2.50000000 2.50000000 	-21.50849288 0.00000000 0.00000000 	1.62346655e-01 0.00000000e+00 0.00000000e+00 	-3.18907192e+01 0.00000000e+00 0.00000000e+00 
H 2.80215144 2.50000000 2.50000000 	21.50849288 0.00000000 0.00000000 	1.62346655e-01 0.00000000e+00 0.00000000e+00 	3.18907192e+01 0.00000000e+00 0.00000000e+00 

temperature: 407.73 K 
kinetic energy: 0.105406 eV 
potential energy: 8.114370 eV 
total energy: 8.219777 eV 
wall time from start: 3.79 s 

Adding atom 0 to the training set.
Uncertainty: [0.16234665 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.2346192387097103
Hyp1 : Length Scale = 0.21777953786979576
Hyp2 : Noise Std = 1e-06
likelihood: 45.95208848954375
likelihood gradient: [ 1.34337054e-04 -3.24601320e-04 -4.00000000e+06]
wall time from start: 3.81 s 
--------------------------------------------------------------------------------
-Frame: 2
Simulation Time: 0.000 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.29362186 2.50000000 2.50000000 	-19.89495491 0.00000000 0.00000000 	6.06330720e-02 0.00000000e+00 0.00000000e+00 	-5.18648464e+01 0.00000000e+00 0.00000000e+00 
H 2.80637814 2.50000000 2.50000000 	19.89495491 0.00000000 0.00000000 	6.06330720e-02 0.00000000e+00 0.00000000e+00 	5.18648464e+01 0.00000000e+00 0.00000000e+00 

temperature: 1078.43 K 
kinetic energy: 0.278795 eV 
potential energy: -2.508449 eV 
total energy: -2.229654 eV 
wall time from start: 3.81 s 
--------------------------------------------------------------------------------
-Frame: 3
Simulation Time: 0.000 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.28747559 2.50000000 2.50000000 	-17.42382815 0.00000000 0.00000000 	2.89212327e-01 0.00000000e+00 0.00000000e+00 	-6.98684224e+01 0.00000000e+00 0.00000000e+00 
H 2.81252441 2.50000000 2.50000000 	17.42382815 0.00000000 0.00000000 	2.89212327e-01 0.00000000e+00 0.00000000e+00 	6.98684224e+01 0.00000000e+00 0.00000000e+00 

temperature: 1957.07 K 
kinetic energy: 0.505942 eV 
potential energy: -2.737955 eV 
total energy: -2.232014 eV 
wall time from start: 3.81 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 3 
wall time from start: 4.11 s 

*-Frame: 3
Simulation Time: 0.000 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.28747559 2.50000000 2.50000000 	-18.03172916 0.00000000 0.00000000 	2.89212327e-01 0.00000000e+00 0.00000000e+00 	-7.01616901e+01 0.00000000e+00 0.00000000e+00 
H 2.81252441 2.50000000 2.50000000 	18.03172916 0.00000000 0.00000000 	2.89212327e-01 0.00000000e+00 0.00000000e+00 	7.01616901e+01 0.00000000e+00 0.00000000e+00 

temperature: 1973.53 K 
kinetic energy: 0.510198 eV 
potential energy: -2.737955 eV 
total energy: -2.227757 eV 
wall time from start: 4.11 s 

Adding atom 0 to the training set.
Uncertainty: [0.28921233 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.5022193369677515
Hyp1 : Length Scale = 0.26639409557176
Hyp2 : Noise Std = 1e-06
likelihood: 70.4841660351524
likelihood gradient: [ 1.77477777e-05 -8.33128579e-05 -6.00000000e+06]
wall time from start: 4.13 s 
--------------------------------------------------------------------------------
-Frame: 4
Simulation Time: 0.000 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.27958952 2.50000000 2.50000000 	-15.78693728 0.00000000 0.00000000 	4.24690501e-02 0.00000000e+00 0.00000000e+00 	-8.64767165e+01 0.00000000e+00 0.00000000e+00 
H 2.82041048 2.50000000 2.50000000 	15.78693728 0.00000000 0.00000000 	4.24690501e-02 0.00000000e+00 0.00000000e+00 	8.64767165e+01 0.00000000e+00 0.00000000e+00 

temperature: 2998.08 K 
kinetic energy: 0.775063 eV 
potential energy: -2.366793 eV 
total energy: -1.591729 eV 
wall time from start: 4.14 s 
--------------------------------------------------------------------------------
-Frame: 5
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.27018024 2.50000000 2.50000000 	-13.58630435 0.00000000 0.00000000 	2.01991514e-01 0.00000000e+00 0.00000000e+00 	-1.00647152e+02 0.00000000e+00 0.00000000e+00 
H 2.82981976 2.50000000 2.50000000 	13.58630435 0.00000000 0.00000000 	2.01991514e-01 0.00000000e+00 0.00000000e+00 	1.00647152e+02 0.00000000e+00 0.00000000e+00 

temperature: 4061.14 K 
kinetic energy: 1.049885 eV 
potential energy: -2.642350 eV 
total energy: -1.592465 eV 
wall time from start: 4.14 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 4 
wall time from start: 4.43 s 

*-Frame: 5
Simulation Time: 0.001 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.27018024 2.50000000 2.50000000 	-13.26095564 0.00000000 0.00000000 	2.01991514e-01 0.00000000e+00 0.00000000e+00 	-1.00490195e+02 0.00000000e+00 0.00000000e+00 
H 2.82981976 2.50000000 2.50000000 	13.26095564 0.00000000 0.00000000 	2.01991514e-01 0.00000000e+00 0.00000000e+00 	1.00490195e+02 0.00000000e+00 0.00000000e+00 

temperature: 4048.48 K 
kinetic energy: 1.046613 eV 
potential energy: -2.642350 eV 
total energy: -1.595737 eV 
wall time from start: 4.43 s 

Adding atom 0 to the training set.
Uncertainty: [0.20199151 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.5932251240836687
Hyp1 : Length Scale = 0.26626547208741974
Hyp2 : Noise Std = 1e-06
likelihood: 95.78697093820287
likelihood gradient: [ 1.14343129e-05  2.05403194e-06 -7.99999998e+06]
wall time from start: 4.46 s 
--------------------------------------------------------------------------------
-Frame: 6
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.25949148 2.50000000 2.50000000 	-10.75629550 0.00000000 0.00000000 	2.88936481e-02 0.00000000e+00 0.00000000e+00 	-1.12076757e+02 0.00000000e+00 0.00000000e+00 
H 2.84050852 2.50000000 2.50000000 	10.75629550 0.00000000 0.00000000 	2.88936481e-02 0.00000000e+00 0.00000000e+00 	1.12076757e+02 0.00000000e+00 0.00000000e+00 

temperature: 5035.89 K 
kinetic energy: 1.301877 eV 
potential energy: -8.970773 eV 
total energy: -7.668896 eV 
wall time from start: 4.46 s 
--------------------------------------------------------------------------------
-Frame: 7
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.24776489 2.50000000 2.50000000 	-8.19461805 0.00000000 0.00000000 	1.39539302e-01 0.00000000e+00 0.00000000e+00 	-1.21219183e+02 0.00000000e+00 0.00000000e+00 
H 2.85223511 2.50000000 2.50000000 	8.19461805 0.00000000 0.00000000 	1.39539302e-01 0.00000000e+00 0.00000000e+00 	1.21219183e+02 0.00000000e+00 0.00000000e+00 

temperature: 5890.98 K 
kinetic energy: 1.522935 eV 
potential energy: -9.192796 eV 
total energy: -7.669861 eV 
wall time from start: 4.46 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 5 
wall time from start: 4.76 s 

*-Frame: 7
Simulation Time: 0.001 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.24776489 2.50000000 2.50000000 	-8.55628393 0.00000000 0.00000000 	1.39539302e-01 0.00000000e+00 0.00000000e+00 	-1.21393660e+02 0.00000000e+00 0.00000000e+00 
H 2.85223511 2.50000000 2.50000000 	8.55628393 0.00000000 0.00000000 	1.39539302e-01 0.00000000e+00 0.00000000e+00 	1.21393660e+02 0.00000000e+00 0.00000000e+00 

temperature: 5907.95 K 
kinetic energy: 1.527322 eV 
potential energy: -9.192796 eV 
total energy: -7.665474 eV 
wall time from start: 4.77 s 

Adding atom 0 to the training set.
Uncertainty: [0.1395393 0.        0.       ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.7488694022424885
Hyp1 : Length Scale = 0.2534629707520455
Hyp2 : Noise Std = 1e-06
likelihood: 120.23726707101737
likelihood gradient: [ 1.88441586e-04 -3.09477311e-03 -9.99999958e+06]
wall time from start: 4.81 s 
--------------------------------------------------------------------------------
-Frame: 8
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.23521275 2.50000000 2.50000000 	-6.56778058 0.00000000 0.00000000 	3.57589437e-02 0.00000000e+00 0.00000000e+00 	-1.28689912e+02 0.00000000e+00 0.00000000e+00 
H 2.86478725 2.50000000 2.50000000 	6.56778058 0.00000000 0.00000000 	3.57589437e-02 0.00000000e+00 0.00000000e+00 	1.28689912e+02 0.00000000e+00 0.00000000e+00 

temperature: 6639.47 K 
kinetic energy: 1.716436 eV 
potential energy: -5.109334 eV 
total energy: -3.392898 eV 
wall time from start: 4.81 s 
--------------------------------------------------------------------------------
-Frame: 9
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.22202691 2.50000000 2.50000000 	-5.01528334 0.00000000 0.00000000 	1.76774576e-01 0.00000000e+00 0.00000000e+00 	-1.34277891e+02 0.00000000e+00 0.00000000e+00 
H 2.87797309 2.50000000 2.50000000 	5.01528334 0.00000000 0.00000000 	1.76774576e-01 0.00000000e+00 0.00000000e+00 	1.34277891e+02 0.00000000e+00 0.00000000e+00 

temperature: 7228.59 K 
kinetic energy: 1.868735 eV 
potential energy: -5.260788 eV 
total energy: -3.392053 eV 
wall time from start: 4.81 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 6 
wall time from start: 5.10 s 

*-Frame: 9
Simulation Time: 0.001 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.22202691 2.50000000 2.50000000 	-4.65786373 0.00000000 0.00000000 	1.76774576e-01 0.00000000e+00 0.00000000e+00 	-1.34105462e+02 0.00000000e+00 0.00000000e+00 
H 2.87797309 2.50000000 2.50000000 	4.65786373 0.00000000 0.00000000 	1.76774576e-01 0.00000000e+00 0.00000000e+00 	1.34105462e+02 0.00000000e+00 0.00000000e+00 

temperature: 7210.04 K 
kinetic energy: 1.863939 eV 
potential energy: -5.260788 eV 
total energy: -3.396849 eV 
wall time from start: 5.10 s 

Adding atom 0 to the training set.
Uncertainty: [0.17677458 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.6903395905888203
Hyp1 : Length Scale = 0.23366385415636054
Hyp2 : Noise Std = 1e-06
likelihood: 145.16385077523907
likelihood gradient: [ 5.89480260e-06  2.59339868e-06 -1.19999978e+07]
wall time from start: 5.16 s 
--------------------------------------------------------------------------------
-Frame: 10
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.20839165 2.50000000 2.50000000 	-2.95872797 0.00000000 0.00000000 	3.99493538e-02 0.00000000e+00 0.00000000e+00 	-1.37779909e+02 0.00000000e+00 0.00000000e+00 
H 2.89160835 2.50000000 2.50000000 	2.95872797 0.00000000 0.00000000 	3.99493538e-02 0.00000000e+00 0.00000000e+00 	1.37779909e+02 0.00000000e+00 0.00000000e+00 

temperature: 7610.56 K 
kinetic energy: 1.967481 eV 
potential energy: -10.911757 eV 
total energy: -8.944276 eV 
wall time from start: 5.16 s 
--------------------------------------------------------------------------------
-Frame: 11
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.19447093 2.50000000 2.50000000 	-1.20828990 0.00000000 0.00000000 	1.91435760e-01 0.00000000e+00 0.00000000e+00 	-1.39790190e+02 0.00000000e+00 0.00000000e+00 
H 2.90552907 2.50000000 2.50000000 	1.20828990 0.00000000 0.00000000 	1.91435760e-01 0.00000000e+00 0.00000000e+00 	1.39790190e+02 0.00000000e+00 0.00000000e+00 

temperature: 7834.26 K 
kinetic energy: 2.025313 eV 
potential energy: -10.970111 eV 
total energy: -8.944799 eV 
wall time from start: 5.17 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 7 
wall time from start: 5.45 s 

*-Frame: 11
Simulation Time: 0.001 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.19447093 2.50000000 2.50000000 	-1.72731595 0.00000000 0.00000000 	1.91435760e-01 0.00000000e+00 0.00000000e+00 	-1.40040582e+02 0.00000000e+00 0.00000000e+00 
H 2.90552907 2.50000000 2.50000000 	1.72731595 0.00000000 0.00000000 	1.91435760e-01 0.00000000e+00 0.00000000e+00 	1.40040582e+02 0.00000000e+00 0.00000000e+00 

temperature: 7862.35 K 
kinetic energy: 2.032575 eV 
potential energy: -10.970111 eV 
total energy: -8.937537 eV 
wall time from start: 5.45 s 

Adding atom 0 to the training set.
Uncertainty: [0.19143576 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.6915997309495541
Hyp1 : Length Scale = 0.21570260044051914
Hyp2 : Noise Std = 1e-06
likelihood: 169.04851081517128
likelihood gradient: [-7.31605207e-02 -1.65852279e-03 -1.39999916e+07]
wall time from start: 5.49 s 
--------------------------------------------------------------------------------
-Frame: 12
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.18038354 2.50000000 2.50000000 	-0.71166841 0.00000000 0.00000000 	5.83572992e-02 0.00000000e+00 0.00000000e+00 	-1.41217213e+02 0.00000000e+00 0.00000000e+00 
H 2.91961646 2.50000000 2.50000000 	0.71166841 0.00000000 0.00000000 	5.83572992e-02 0.00000000e+00 0.00000000e+00 	1.41217213e+02 0.00000000e+00 0.00000000e+00 

temperature: 7995.03 K 
kinetic energy: 2.066874 eV 
potential energy: -5.466834 eV 
total energy: -3.399960 eV 
wall time from start: 5.49 s 
--------------------------------------------------------------------------------
-Frame: 13
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.16622748 2.50000000 2.50000000 	-0.15865703 0.00000000 0.00000000 	2.75842949e-01 0.00000000e+00 0.00000000e+00 	-1.41637081e+02 0.00000000e+00 0.00000000e+00 
H 2.93377252 2.50000000 2.50000000 	0.15865703 0.00000000 0.00000000 	2.75842949e-01 0.00000000e+00 0.00000000e+00 	1.41637081e+02 0.00000000e+00 0.00000000e+00 

temperature: 8042.64 K 
kinetic energy: 2.079183 eV 
potential energy: -5.477852 eV 
total energy: -3.398669 eV 
wall time from start: 5.49 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 8 
wall time from start: 5.80 s 

*-Frame: 13
Simulation Time: 0.001 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.16622748 2.50000000 2.50000000 	0.35889377 0.00000000 0.00000000 	2.75842949e-01 0.00000000e+00 0.00000000e+00 	-1.41387401e+02 0.00000000e+00 0.00000000e+00 
H 2.93377252 2.50000000 2.50000000 	-0.35889377 0.00000000 0.00000000 	2.75842949e-01 0.00000000e+00 0.00000000e+00 	1.41387401e+02 0.00000000e+00 0.00000000e+00 

temperature: 8014.31 K 
kinetic energy: 2.071859 eV 
potential energy: -5.477852 eV 
total energy: -3.405993 eV 
wall time from start: 5.80 s 

Adding atom 0 to the training set.
Uncertainty: [0.27584295 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.6133525351493617
Hyp1 : Length Scale = 0.20365935964360024
Hyp2 : Noise Std = 1e-06
likelihood: 193.68982191686825
likelihood gradient: [ 1.04633788e-02 -1.35932053e-01 -1.59999803e+07]
wall time from start: 6.30 s 
--------------------------------------------------------------------------------
-Frame: 14
Simulation Time: 0.001 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.15210606 2.50000000 2.50000000 	1.29783349 0.00000000 0.00000000 	6.62676816e-02 0.00000000e+00 0.00000000e+00 	-1.40588151e+02 0.00000000e+00 0.00000000e+00 
H 2.94789394 2.50000000 2.50000000 	-1.29783349 0.00000000 0.00000000 	6.62676816e-02 0.00000000e+00 0.00000000e+00 	1.40588151e+02 0.00000000e+00 0.00000000e+00 

temperature: 7923.96 K 
kinetic energy: 2.048501 eV 
potential energy: -9.298126 eV 
total energy: -7.249626 eV 
wall time from start: 6.31 s 
--------------------------------------------------------------------------------
-Frame: 15
Simulation Time: 0.002 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.13810985 2.50000000 2.50000000 	2.44748992 0.00000000 0.00000000 	2.80064240e-01 0.00000000e+00 0.00000000e+00 	-1.38781308e+02 0.00000000e+00 0.00000000e+00 
H 2.96189015 2.50000000 2.50000000 	-2.44748992 0.00000000 0.00000000 	2.80064240e-01 0.00000000e+00 0.00000000e+00 	1.38781308e+02 0.00000000e+00 0.00000000e+00 

temperature: 7721.59 K 
kinetic energy: 1.996184 eV 
potential energy: -9.246560 eV 
total energy: -7.250375 eV 
wall time from start: 6.31 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 9 
wall time from start: 6.62 s 

*-Frame: 15
Simulation Time: 0.002 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.13810985 2.50000000 2.50000000 	1.79967583 0.00000000 0.00000000 	2.80064240e-01 0.00000000e+00 0.00000000e+00 	-1.39093830e+02 0.00000000e+00 0.00000000e+00 
H 2.96189015 2.50000000 2.50000000 	-1.79967583 0.00000000 0.00000000 	2.80064240e-01 0.00000000e+00 0.00000000e+00 	1.39093830e+02 0.00000000e+00 0.00000000e+00 

temperature: 7756.40 K 
kinetic energy: 2.005185 eV 
potential energy: -9.246560 eV 
total energy: -7.241375 eV 
wall time from start: 6.62 s 

Adding atom 0 to the training set.
Uncertainty: [0.28006424 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.6138355696265253
Hyp1 : Length Scale = 0.19539130291080367
Hyp2 : Noise Std = 1e-06
likelihood: 217.6270263203563
likelihood gradient: [-5.48148650e-02 -9.96570074e-02 -1.79999539e+07]
wall time from start: 7.02 s 
--------------------------------------------------------------------------------
-Frame: 16
Simulation Time: 0.002 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.12428729 2.50000000 2.50000000 	2.22012399 0.00000000 0.00000000 	7.47186985e-02 0.00000000e+00 0.00000000e+00 	-1.37154572e+02 0.00000000e+00 0.00000000e+00 
H 2.97571271 2.50000000 2.50000000 	-2.22012399 0.00000000 0.00000000 	7.47186985e-02 0.00000000e+00 0.00000000e+00 	1.37154572e+02 0.00000000e+00 0.00000000e+00 

temperature: 7541.63 K 
kinetic energy: 1.949662 eV 
potential energy: -5.111769 eV 
total energy: -3.162107 eV 
wall time from start: 7.02 s 
--------------------------------------------------------------------------------
-Frame: 17
Simulation Time: 0.002 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.11067894 2.50000000 2.50000000 	2.29130067 0.00000000 0.00000000 	3.11574527e-01 0.00000000e+00 0.00000000e+00 	-1.34978140e+02 0.00000000e+00 0.00000000e+00 
H 2.98932106 2.50000000 2.50000000 	-2.29130067 0.00000000 0.00000000 	3.11574527e-01 0.00000000e+00 0.00000000e+00 	1.34978140e+02 0.00000000e+00 0.00000000e+00 

temperature: 7304.18 K 
kinetic energy: 1.888276 eV 
potential energy: -5.049409 eV 
total energy: -3.161133 eV 
wall time from start: 7.03 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 10 
wall time from start: 7.33 s 

*-Frame: 17
Simulation Time: 0.002 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.11067894 2.50000000 2.50000000 	2.77308409 0.00000000 0.00000000 	3.11574527e-01 0.00000000e+00 0.00000000e+00 	-1.34745715e+02 0.00000000e+00 0.00000000e+00 
H 2.98932106 2.50000000 2.50000000 	-2.77308409 0.00000000 0.00000000 	3.11574527e-01 0.00000000e+00 0.00000000e+00 	1.34745715e+02 0.00000000e+00 0.00000000e+00 

temperature: 7279.05 K 
kinetic energy: 1.881779 eV 
potential energy: -5.049409 eV 
total energy: -3.167630 eV 
wall time from start: 7.33 s 

Adding atom 0 to the training set.
Uncertainty: [0.31157453 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.6139633868954473
Hyp1 : Length Scale = 0.19272286800898408
Hyp2 : Noise Std = 1e-06
likelihood: 242.51329042836986
likelihood gradient: [-1.17480402e-01 -1.85570124e-01 -1.99999036e+07]
wall time from start: 7.51 s 
--------------------------------------------------------------------------------
-Frame: 18
Simulation Time: 0.002 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.09733815 2.50000000 2.50000000 	3.23610669 0.00000000 0.00000000 	6.20508099e-02 0.00000000e+00 0.00000000e+00 	-1.31846721e+02 0.00000000e+00 0.00000000e+00 
H 3.00266185 2.50000000 2.50000000 	-3.23610669 0.00000000 0.00000000 	6.20508099e-02 0.00000000e+00 0.00000000e+00 	1.31846721e+02 0.00000000e+00 0.00000000e+00 

temperature: 6969.21 K 
kinetic energy: 1.801679 eV 
potential energy: -8.270761 eV 
total energy: -6.469082 eV 
wall time from start: 7.51 s 
--------------------------------------------------------------------------------
-Frame: 19
Simulation Time: 0.002 ps 
El 			  Position (A) 					 GP Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.08430960 2.50000000 2.50000000 	3.87619920 0.00000000 0.00000000 	2.54764407e-01 0.00000000e+00 0.00000000e+00 	-1.28415555e+02 0.00000000e+00 0.00000000e+00 
H 3.01569040 2.50000000 2.50000000 	-3.87619920 0.00000000 0.00000000 	2.54764407e-01 0.00000000e+00 0.00000000e+00 	1.28415555e+02 0.00000000e+00 0.00000000e+00 

temperature: 6611.20 K 
kinetic energy: 1.709126 eV 
potential energy: -8.178729 eV 
total energy: -6.469603 eV 
wall time from start: 7.52 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 11 
wall time from start: 7.82 s 

*-Frame: 19
Simulation Time: 0.002 ps 
El 			  Position (A) 					 DFT Force (ev/A) 						 Std. Dev (ev/A) 							 Velocities (A/ps) 
H 2.08430960 2.50000000 2.50000000 	3.41127024 0.00000000 0.00000000 	2.54764407e-01 0.00000000e+00 0.00000000e+00 	-1.28639849e+02 0.00000000e+00 0.00000000e+00 
H 3.01569040 2.50000000 2.50000000 	-3.41127024 0.00000000 0.00000000 	2.54764407e-01 0.00000000e+00 0.00000000e+00 	1.28639849e+02 0.00000000e+00 0.00000000e+00 

temperature: 6634.31 K 
kinetic energy: 1.715101 eV 
potential energy: -8.178729 eV 
total energy: -6.463628 eV 
wall time from start: 7.82 s 

Adding atom 0 to the training set.
Uncertainty: [0.25476441 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.6346010919829839
Hyp1 : Length Scale = 0.18999157812731896
Hyp2 : Noise Std = 1e-06
likelihood: 267.0731074415073
likelihood gradient: [ 4.45316757e-02 -1.25831433e+00 -2.19997906e+07]
wall time from start: 8.61 s 
▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
Run complete. 
